### PR TITLE
Fix config (config on bedrock.start now).

### PR DESCRIPTION
**Pull request by dmitrizagidulin**
Adding support for did:key and did:v1 DIDs, fix config eventing.

### CODE REVIEW

File: lib/config.js [hunk 1]
```diff
@@ -6,10 +6,25 @@
...
+const {CryptoLD} = require('crypto-ld');
+
+const cryptoLd = new CryptoLD();
+cryptoLd.use(Ed25519VerificationKey2020);
 
 const namespace = 'vc-issuer';
 const cfg = config[namespace] = {};
 
+// bedrock-did-io configuration
+config['did-io'].methods.key.verificationSuite = Ed25519VerificationKey2020;
```
Comment (dlongley) [inline lib/config.js:20]: bedrock-did-io needs to be required if it will be modified here.

File: lib/config.js [hunk 2]
```diff
@@ -6,10 +6,25 @@
...
+
+config['did-io'].methods.v1.verificationSuite = Ed25519VerificationKey2020;
+config['did-io'].methods.v1.cryptoLd = cryptoLd;
+
+// allow self-signed certificates in dev
+config['https-agent'].rejectUnauthorized = false;
```
Comment (dlongley) [inline lib/config.js:26]: I think we have a different pattern/practice for setting this -- we don't typically set it in default configs in modules. Can this be checked?

File: lib/config.js [hunk 3]
```diff
@@ -6,10 +6,25 @@
...
+cryptoLd.use(Ed25519VerificationKey2020);
 
 const namespace = 'vc-issuer';
 const cfg = config[namespace] = {};
 
+// bedrock-did-io configuration
+config['did-io'].methods.key.verificationSuite = Ed25519VerificationKey2020;
+
+config['did-io'].methods.v1.verificationSuite = Ed25519VerificationKey2020;
+config['did-io'].methods.v1.cryptoLd = cryptoLd;
```
Comment (dlongley) [inline lib/config.js:23]: I suspect we want to set all of these things at the app level, not here. This creates a tight binding and introduces other dependencies that make this module harder to maintain.

**Review by dlongley — CHANGES_REQUESTED** [review]

**Comment by dmitrizagidulin** [comment]
Closing the PR, moving this config to the top level app.

**Pull request closed without merging**